Makefile

```diff
@@ -291,9 +291,10 @@ add-copyright:
 .windows-test-unit: .coverage
 	@echo
 	@echo "==> Running unit tests <=="
-	CGO_ENABLED=1 $(GO) test $(TEST_ARGS) -race -coverpkg=antrea.io/antrea/cmd/...,antrea.io/antrea/pkg/... \
+	@pkgs=$$($(GO) list antrea.io/antrea/cmd/... antrea.io/antrea/pkg/... | grep -v antrea.io/antrea/pkg/agent/controller | grep -v antrea.io/antrea/pkg/controller); \
```

Comment:
excluding `antrea.io/antrea/pkg/agent/controller` on Windows does not seem right to me?